.github/workflows/check_examples.yaml

```diff
@@ -22,6 +22,9 @@ jobs:
   check:
     runs-on: ubuntu-latest
     strategy:
+      # Don't cancel all in-progress and queued jobs in the matrix if any job in the matrix fails.
+      # That will be helpful to catch any issues related to a particular Kubernetes version.
+      fail-fast: false
```

Comment:
Ideally we would have this as an input variable. Sometimes it's useful to fail faster when iterating towards a green run, fixing one failure at a time and re-running.

Reply:
Thanks, I made it because yesterday GH was unstable and sometimes couldn't get Terraform so no tests were executed. Made this change to increase the chance to get at least one working scenario. :) Revert it back now.